src/test/java/service/JavaQuestionServiceTest.java

```diff
@@ -0,0 +1,49 @@
+package service;
+
+import dto.Question;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import services.JavaQuestionServiceImpl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+public class JavaQuestionServiceTest {
+    @Mock
+    private JavaQuestionServiceImpl serviceMock;
+
+    @BeforeEach
+    public void setUp() {
+        serviceMock = new JavaQuestionServiceImpl();
+    }
```

Comment:
Это всё можно заменить на

```suggestion
    private JavaQuestionServiceImpl serviceMock = new JavaQuestionServiceImpl();
```